src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java

```diff
@@ -104,11 +104,6 @@ public void testCreateDeleteFeedbackQuestion() throws InvalidParametersException
         AssertionError ae = assertThrows(AssertionError.class, () -> fqDb.createEntity(null));
         AssertHelper.assertContains(Const.StatusCodes.DBLEVEL_NULL_INPUT, ae.getLocalizedMessage());
 
-        ______TS("invalid params");
```

Comment:
I would suggest to have another invalid params (e.g. empty courseid, empty questionDetails etc.) instead of removing the test.